grammar.js

```diff
@@ -93,7 +93,7 @@ module.exports = grammar({
         $.field_declaration_list_body,
         "}",
       ),
-
+    // FIXME PLS
```

Comment:
This will need some reworking as I didn't originally make it so that fields could just have a singular expression. This would involve a significant number of changes

Reply:
> This will need some reworking as I didn't originally make it so that blocks could just have a singular expression. This would involve a significant number of changes

Sorry, it is just a mark, I foget to delete. If someting has not be completed, I will be glade to help